static/components/f-hidden.js

```diff
@@ -8,7 +8,11 @@ customElements.define(
       const { schema, value, namePrefix } = this;
       const name = `${namePrefix}.${schema.name}`;
 
-      push(this, "input", { type: "hidden", name, value });
+      push(this, "input", {
+        type: "hidden",
+        name,
+        value: value || schema.value,
```

Comment:
This would assign the `schema.value` if the current value can be evaluated as false, like `0`.
Probably `value ?? schema.value` is better because it only check for `null` or `undefined`values.